From: Michael Fladischer <FladischerMichael@fladi.at>
Date: Sun, 30 Jul 2017 22:22:36 +0200
Subject: Disable intershpinx mapping for now.

The best way to fix network requests when building the documenation would be to
patch INTERSPHINX_MAPPING in sphinx_celery.conf in src:sphinx-celery.
---
 docs/conf.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/conf.py b/docs/conf.py
index a807011..774837b 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -28,4 +28,5 @@ globals().update(conf.build_config(
         'kombu.utils',
         'kombu.transport.virtual.base',
     ],
+    intersphinx_mapping={},
 ))
